cwms-http-client/src/main/java/mil/army/usace/hec/cwms/http/client/HttpDownloader.java

```diff
@@ -0,0 +1,32 @@
+package mil.army.usace.hec.cwms.http.client;
+import mil.army.usace.hec.cwms.http.client.request.HttpRequestExecutor;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+
+public class HttpDownloader {
```

Comment:
I don't think this should be a public class (and these functions should probably just exist in the TextTimeSeriesController) as I don't see any external use-cases for these methods.